code-of-conduct.md

```diff
@@ -37,6 +37,8 @@ Examples of unacceptable behavior by participants include:
 * Publishing others' private information, such as a physical or electronic
   address, without explicit permission. This includes any sort of "outing" of
   of any aspect of someone's identity without their consent.
+* Publishing screenshots or quotes, specially from identity channels, without a user's explicit consent.
```

Comment:
the intention behind that emphasis is to clarify that enforcement will probably be harsher/more aggressive for those comms. Do you have a better wording that you'd suggest? I had trouble constructing this clause.